modules/xmpp/SDPUtil.js

```diff
@@ -443,6 +443,30 @@ var SDPUtil = {
             .split(" ")
             .map(ssrcStr => parseInt(ssrcStr));
     },
+
+    /**
+     * Sets the given codecName as the preferred codec by
+     *  moving it to the beginning of the payload types
+     *  list (modifies the given mline in place)
+     * @param {object} videoMLine the video mline object from
+     *  an sdp as parsed by transform.parse
+     * @param {string} the name of the preferred codec
+     */
+    preferVideoCodec: function(videoMLine, codecName) {
+        let payloadType = null;
+        videoMLine.rtp.forEach(rtp => {
+            if (rtp.codec === codecName) {
+                payloadType = rtp.payload;
+            }
+        });
+        if (payloadType) {
+            let payloadTypes = videoMLine.payloads.split(" ").map(p => parseInt(p));
```

Comment:
yes i do understand that, lyubo (which is why i said it in my note).  just a little off-hand commentary.  changed to const.